src/MasterPeace/Bundle/QuizBundle/Entity/QuizResult.php

```diff
@@ -25,23 +24,9 @@ class QuizResult
     private $id;
 
     /**
-     * @var Answer
-     *
-     * @ORM\ManyToOne(targetEntity="MasterPeace\Bundle\BookBundle\Entity\Answer")
-     */
-    private $answer;
-
-    /**
-     * @var Question
-     *
-     * @ORM\ManyToOne(targetEntity="MasterPeace\Bundle\BookBundle\Entity\Question")
-     */
-    private $question;
-
-    /**
      * @var User
      *
-     * @ORM\ManyToOne(targetEntity="MasterPeace\Bundle\UserBundle\Entity\User")
+     * @ORM\ManyToMany(targetEntity="MasterPeace\Bundle\UserBundle\Entity\User")
```

Comment:
jei many to many, tai vienas quiz_rezultas tures daug studentu? ta prasme daug studentu gali vos ne ta pati quiz spresti